scripts/run_backend_tests.py

```diff
@@ -73,6 +73,8 @@
     os.path.join(common.OPPIA_TOOLS_DIR, 'psutil-%s' % common.PSUTIL_VERSION),
     os.path.join(
         common.OPPIA_TOOLS_DIR, 'PyGithub-%s' % common.PYGITHUB_VERSION),
+    os.path.join(
```

Comment:
I think something is wrong with your setup here. Why are you still installing this in oppia_tools?

Please fix this -- I tried running your PR on my local machine and it gave a 500 server error. You can test by checking out a copy of Oppia in a second dir on your local machine and performing a full install from scratch -- make sure the start script results in the page loading without any issues.

Reply:
> I think something is wrong with your setup here. Why are you still installing this in oppia_tools?

I saw that the other libs which are installed using pip are installed in oppia_tools. I'll change it and add the link in `manifest.json` instead.

> I tried running your PR on my local machine and it gave a 500 server error.

It was giving me a 500 error as well which is why I had to insert the path in a bunch of places. It works for me now, still, I'll double-check it in a fresh copy.

Reply:
Remember: oppia_tools is for stuff that is not uploaded to prod. It's for developer scripts, and stuff that doesn't already exist in App Engine. If you want something to be in prod, then it needs to go in third_party.

Is the library not on github? You could install from there instead perhaps.

Reply:
I've moved this from oppia_tools to third_party_libs and I've tested it locally on a fresh copy of the repo.